resources/views/partials/footer-partials.blade.php

```diff
@@ -1,34 +1,20 @@
 <section id="footer" style="background-color:#32CD99">
     <div class="container">
-        <div class="row text-center text-xs-center text-sm-left text-md-left">
-            <div class="col-xs-10 offset-5">
-                <h5>Quick links</h5>
-                <ul class="list-unstyled quick-links">
-                    <li><a href="javascript:void();"><i class="fa fa-asterisk"></i>Home</a></li>
-                    <li><a href="javascript:void();"><i class="fa fa-asterisk"></i>About</a></li>
-                    <li><a href="javascript:void();"><i class="fa fa-asterisk"></i>FAQ</a></li>
-                    <li><a href="javascript:void();"><i class="fa fa-asterisk"></i>Get Started</a></li>
-                    <li><a href="javascript:void();"><i class="fa fa-asterisk"></i>Videos</a></li>
-                </ul>
-            </div>
-        </div>
-        <div class="row">
-            <div class="col-xs-12 col-sm-12 col-md-12">
-                <ul class="list-unstyled list-inline social text-center">
-                    <li class="list-inline-item"><a href="javascript:void();"><i class="fa fa-facebook"></i></a></li>
-                    <li class="list-inline-item"><a href="javascript:void();"><i class="fa fa-twitter"></i></a></li>
-                    <li class="list-inline-item"><a href="javascript:void();"><i class="fa fa-instagram"></i></a></li>
-                    <li class="list-inline-item"><a href="javascript:void();"><i class="fa fa-google-plus"></i></a></li>
-                    <li class="list-inline-item"><a href="javascript:void();" target="_blank"><i class="fa fa-envelope"></i></a></li>
-                </ul>
-            </div>            
-        </div>
         <div class="row">
             <div class="col-xs-6 col-sm-6 col-md-6 text-white">
-                <p style="color: white;">Powered By <span>Female Bootcamp</span></p>
-                <p class="h6" style="color: white;">&copy All right Reserved.<span> Adeva</span></p>
+                <p style="color: white;">Powered By <a target="_blank"
+                        href="https://femalebootcamp.adevait.com/"><span>Female
+                            Bootcamp</span></a></p>
+                <p class="h6" style="color: white;">&copy All right Reserved.<a target="_blank"
+                        href="https://adevait.com/"><span>
+                            Adeva</span></a></p>
+            </div>
+            <div class="col-xs-6 col-sm-6 col-md-6 text-right">
+                <a href="{{route('team')}}">
+                    <h2><span>Meet The Team</span></h2>
```

Comment:
Please avoid using `<h2>` for text other than headings. In this case you can use `<span>` or `<p>` and set the font-size in css.